security/certs.go

```diff
@@ -129,7 +142,7 @@ func RunCreateCACert(certsDir string, keySize int) error {
 // RunCreateNodeCert is the entry-point from the command-line interface
 // to generate node certs and keys:
 // - node.server.{crt,key}: server cert with list of dns/ip addresses
-// - node.{crt,key}: client cert with "node" as the Common Name.
+// - node.client.{crt,key}: client cert with "node" as the Common Name.
```

Comment:
agreed, although this may be more complexity than we want. We'll already need special permissions for "node" vs "root". To really identity nodes on incoming requests, we should log the node ID or address.